filament/src/RenderTarget.cpp

```diff
@@ -28,7 +28,7 @@ namespace filament {
 using namespace backend;
 
 struct RenderTarget::BuilderDetails {
-    FRenderTarget::Attachment mAttachments[RenderTarget::ATTACHMENT_COUNT];
+    FRenderTarget::Attachment mAttachments[RenderTarget::ATTACHMENT_COUNT] = {};
```

Comment:
Was this necessary?  If so I don't totally understand how C++ field initializers work.

Reply:
An array is left uninitialized otherwise.

Reply:
But `Attachment` has field initializers.  Just now did a test in glot.io:

https://glot.io/snippets/fwstbhey6q

Reply:
yeah, that seems strange that it's needed. I don't think it'll hurt too much anyways, since we already have a default initializer.